addons/website/models/website.py

```diff
@@ -243,10 +243,14 @@ def _compute_menu(self):
     def _compute_blocked_third_party_domains(self):
         for website in self:
             custom_list = website.sudo().custom_blocked_third_party_domains
+
+            full_list = DEFAULT_BLOCKED_THIRD_PARTY_DOMAINS
             if custom_list:
-                full_list = f'{DEFAULT_BLOCKED_THIRD_PARTY_DOMAINS}\n{custom_list}'
-            else:
-                full_list = DEFAULT_BLOCKED_THIRD_PARTY_DOMAINS
+                first = custom_list.splitlines()[0].split(' ')[0].strip()
```

Comment:
Why do you need the `.split(' ')[0]` ?

Reply:
(btw I supposed I am qsaodoo that you pinged ^^)

Reply:
be sure to only take inly the first word of the firs tline

```
#ignore_default //use our own list of 3rd party
youtube.com
```

Reply:
A javascript comment after a python comment? ^^ The strip after split on spaces was useless though and `     #ignore_default` would not have worked -> But sure, I used `startswith` instead.